templates/zerver/emails/notify_change_in_email.html

```diff
@@ -1,12 +1,23 @@
 {% extends "zerver/emails/email_base.html" %}
 
+{% block illustration %}
+<img src="{{ realm_uri }}/static/generated/emails/logo.svg"/>
+{% endblock %}
+
 {% block content %}
 <p>Hi,</p>
 <p>
-    We just wanted to let you know that the email associated with your Zulip account was recently changed to <a href="{{ new_email }}" style="color:#08c">{{ new_email }}</a>. If you did not request this change, please contact us immediately at <a href="mailto:{{ support_email }}" style="color:#08c">{{ support_email }}</a>.
+    We just wanted to let you know that the email associated with your Zulip account was recently changed to <a href="{{ new_email }}">{{ new_email }}</a>. If you did not request this change, please contact us immediately at <a href="mailto:{{ support_email }}">{{ support_email }}</a>.
 </p>
 <p>
     Best,<br />
     The Zulip Team
 </p>
 {% endblock %}
+
+{% block manage_preferences %}
```

Comment:
As with password reset emails, we probably don't want an unsubscribe link here because this is a one-off user-requested email related to their account security.